frontend/src/components/app/App.js

```diff
@@ -27,26 +30,20 @@ function App() {
         <label className="button-label" htmlFor="second-city-select">
           Add Second City
         </label>
+        {searchClicked ? (
+        <>
+          <FlightForm className="flightForm" id="firstCity" />
+          <FlightForm className="flightForm" id="secondCity" />
+          <label className="button-label" htmlFor="second-city-select">
+            Remove Second City
+          </label>
+        </>
+      ) : (
         <FlightForm className="flightForm" id="firstCity" />
-      </div>
-    );
-  } else {
-    return (
-      <div className="App">
```

Comment:
I'm not fully sure why this has been removed? Or if this is an older version. I know @amh4 and @ShaunFlood have been looking after the two forms so it might be worth them checking